package.json

```diff
@@ -3,14 +3,17 @@
 	"repository": "https://github.com/liferay/clay",
 	"size-limit": [
 		{
-			"path": "lib/all.js",
-			"limit": "150 kb"
+			"path": "all.js",
+			"limit": "140 kb",
+			"ignore": [
```

Comment:
What does "ignore" here means exactly in the context of `size-limit`?

Reply:
@jbalsas 

It removes the dependency from the generated size. For example, if we don't have these two ignored we get this for node_modules
![Screen Shot 2020-05-13 at 10 33 13 AM](https://user-images.githubusercontent.com/6843530/81845333-45676300-9505-11ea-819a-028c11810075.png)


Whereas with them ignored we get
![Screen Shot 2020-05-13 at 10 32 36 AM](https://user-images.githubusercontent.com/6843530/81845285-3aacce00-9505-11ea-9873-ac2c942437ed.png)

Reason behind ignoring them is because react and react-dom are peer dependencies of these packages.

Reply:
Ok, that works for now... but we need to be careful about marketing if it ever comes to that, since it might feel like we're hiding needed deps. 

If you're bringing in Clay to a React project, then by all means, this is the library footprint. But to start from scratch, you'd need to throw in some additional bytes.

Let's focus on that tiny `moment` problem, then :D